src/unitxt/metrics.py

```diff
@@ -1265,6 +1266,10 @@ class TokenOverlap(InstanceMetric):
     def compute(
         self, references: List[Any], prediction: Any, additional_inputs: List[Dict]
     ) -> dict:
+        import ast
+
+        references = ast.literal_eval(references)
```

Comment:
Why is this needed?

Reply:
I might be doing something wrong but references were definitely a string 
```python
"['reference1',...]"
```

Reply:
I think the issue is that RAG task you defined as a field answers (which is a list of possible answers), but you used InputOutputTemplate and not MultiReferenceTemplate.  So the list of answers was converted to a string representation.

@elronbandel - we need to better deal with this.

Reply:
even when I am using MultiReferenceTemplate I have the same problem
note that when I test the metric I am not facing it, only when I test it within the card testing